birdhouse/components/thredds/config/magpie/providers.cfg.template

```diff
@@ -33,6 +33,7 @@ providers:
           dodsC,
           wcs,
           wms,
-          ncss,
```

Comment:
Leave the old location.
`THREDDS_VERSION` can be overridden, so a server could still employ the older variant. 

Given that, that will most probably cause a conflict in the `data_type` resolution order if all 3 are defined, so maybe alternate paths or a dynamic variable resolution will be needed here.

Reply:
ugh... you're right. Ok I'll figure something out.